### PR TITLE
games-util/springlobby: 0.269 version bump

### PR DESCRIPTION
**Pull request by OursDesCavernes**
Closes: https://bugs.gentoo.org/708400
Package-Manager: Portage-2.3.89, Repoman-2.3.20

### CODE REVIEW

File: games-util/springlobby/springlobby-0.269.ebuild [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+WX_GTK_VER="3.0"
+inherit cmake-utils gnome2-utils wxwidgets
```
Comment (juippis) [inline games-util/springlobby/springlobby-0.269.ebuild]: Use *cmake.eclass*.

Comment (juippis) [inline games-util/springlobby/springlobby-0.269.ebuild]: Looks like *gnome2-utils* is only used to update icon cache, it shouldn't even work with EAPI-7. You should use *xdg-utils* and adjust the `icon_cache_update` calls below.

File: games-util/springlobby/springlobby-0.269.ebuild [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+HOMEPAGE="https://springlobby.springrts.com"
+SRC_URI="https://springlobby.springrts.com/dl/stable/${P}.tar.bz2"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug +libnotify +nls +sound"
+
+RDEPEND="
+	>=dev-libs/boost-1.35:=
```
Comment (juippis) [inline games-util/springlobby/springlobby-0.269.ebuild]: The version requirement can be dropped for a cleaner ebuild, it's always satisfied in ::gentoo tree.

File: games-util/springlobby/springlobby-0.269.ebuild [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+	dev-libs/openssl:0=
+	net-misc/curl
+	sys-libs/zlib[minizip]
+	x11-libs/libICE
+	x11-libs/libSM
+	x11-libs/libXext
+	x11-libs/wxGTK:${WX_GTK_VER}[X]
+	libnotify? ( x11-libs/libnotify )
+	sound? ( media-libs/alure
```
Comment (juippis) [inline games-util/springlobby/springlobby-0.269.ebuild]: Please multi-line this properly.

File: games-util/springlobby/springlobby-0.269.ebuild [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+		media-libs/openal
+	)
+"
+
+DEPEND="${RDEPEND}
+	nls? ( sys-devel/gettext )
```
Comment (juippis) [inline games-util/springlobby/springlobby-0.269.ebuild]: With EAPI-7 this dependency is a BDEPEND instead.

File: games-util/springlobby/springlobby-0.269.ebuild [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+		-DOPTION_NOTIFY=$(usex libnotify)
+		-DOPTION_SOUND=$(usex sound)
+		-DOPTION_TRANSLATION_SUPPORT=$(usex nls)
+		-DAUX_VERSION="(Gentoo,${ARCH})"
+		)
+	cmake-utils_src_configure
+}
+
+pkg_postinst() {
+	gnome2_icon_cache_update
```
Comment (juippis) [inline games-util/springlobby/springlobby-0.269.ebuild]: `xdg_icon_cache_update`

File: games-util/springlobby/springlobby-0.269.ebuild [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+}
+
+pkg_postrm() {
+	gnome2_icon_cache_update
```
Comment (juippis) [inline games-util/springlobby/springlobby-0.269.ebuild]: `xdg_icon_cache_update`

**Comment by OursDesCavernes** [comment]
Thanks for the reviews, I fixed all the comments

**Comment by juippis** [comment]
Hmm, but it doesn't seem like your fixes made it all the way here :(

The ebuild I see is still exactly the same I reviewed earlier.

**Comment by OursDesCavernes** [comment]
My bad, git commit works better when you add the files :upside_down_face: 
It should be ok now

**Pull request closed without merging**